pages/docs/dev-docs/dev-development.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Development setup
+keywords: pages, development
+permalink: dev-docs-dev-development.html
```

Comment:
Why do we need a new page, if we already have the (stub) `dev-docs-overview.html`?

Reply:
Hmm the more I think about it, the more it makes sense

Reply:
Moved it to overview